scripts/test

```diff
@@ -35,26 +37,22 @@ then
 
         echo "Updating Scala dependencies"
         docker-compose \
-            run --rm --no-deps api-server update
+            run --rm --no-deps sbt \
+            update
 
         echo "Linting Scala source code"
         docker-compose \
-            run --rm --no-deps api-server scalafmtCheck
-        docker-compose \
-            run --rm --no-deps api-server scalafmtSbtCheck
-        docker-compose \
-            run --rm --no-deps api-server scapegoat
+            run --rm --no-deps sbt \
+            ";scalafmtCheck;scalafmtSbtCheck;scapegoat"
```

Comment:
This is to avoid the overhead of starting the container multiple times.

Reply:
Nice, this is a good optimization